src/main/flight/mixer.c

```diff
@@ -887,6 +913,14 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensa
     updateDynLpfCutoffs(currentTimeUs, throttle);
 #endif
 
+#if defined(USE_THRUST_LINEARIZATION)
+    // reestablish old throttle stick feel by counter compensating thrust linearization
+    if (motorConfig()->thrustLinearization) {
+        const float lt = motorConfig()->thrustLinearization * 0.01f;
```

Comment:
this may be moved into function similar to applyThrustLinearization - changing linearization in future will be easier

Reply:
done